config.default

```diff
@@ -43,6 +43,11 @@ parallelTaskCount = 1
 # Default is no prefix (run_analysis.py is executed directly)
 commandPrefix = 
 
+# the parallelism mode in ncclimo ("serial" or "bck")
+# Set this to "bck" if running on a machine that can handle 12 simultaneous
```

Comment:
you are right. Could you just add `("background parallelism")` right after `"bck"`, for clarity?